app/fissile_test.go

```diff
@@ -417,7 +406,7 @@ func TestGenerateAuth(t *testing.T) {
 		},
 		`auth/auth-role-pointless.yaml`: []string{
 			`{
-				"apiVersion": "rbac.authorization.k8s.io/v1beta1",
+				"apiVersion": "rbac.authorization.k8s.io/v1",
```

Comment:
Does this imply a minimum version of kube will be needed to handle the output ?
If yes, are we documenting this version somewhere ?

Reply:
We're already using `rbac.authorization.k8s.io/v1` for [`ClusterRoleBinding`](https://github.com/cloudfoundry-incubator/fissile/blob/c617cf6372fe75ebe255f2920c1dff6d6d0059ba/kube/rbac.go#L41), so this isn't actually a new requirement.  It's supported at least as far back as [Kubernetes 1.9](https://v1-9.docs.kubernetes.io/docs/reference/generated/kubernetes-api/v1.9/#clusterrolebindinglist-v1-rbac) (and docs for older versions are busted).

Reply:
ok